spec/CloudCode.spec.js

```diff
@@ -2963,9 +2996,69 @@ describe('afterLogin hook', () => {
     Parse.Cloud.afterSave('TestObject2', (req) => {
       expect(req.context.a).toEqual('a');
     });
-    const obj = new Parse.Object("TestObject");
-    const obj2 = new Parse.Object("TestObject2");
-    obj.set("obj2", obj2);
+    const obj = new Parse.Object('TestObject');
+    const obj2 = new Parse.Object('TestObject2');
+    obj.set('obj2', obj2);
     await obj.save(null, { context: { a: 'a' } });
   });
+
+  it('should have access to context as saveAll argument', async () => {
+    // Declare triggers
+    Parse.Cloud.beforeSave('TestObject', (req) => {
+      expect(req.context.a).toEqual('a');
+    });
+    Parse.Cloud.afterSave('TestObject', (req) => {
+      expect(req.context.a).toEqual('a');
+    });
+    // Save object
+    const obj = new TestObject();
+    await Parse.Object.saveAll([obj], { context: { a: 'a' } });
+  });
+
+  it('should have access to context as destroyAll argument', async () => {
+    // Declare triggers
+    Parse.Cloud.beforeDelete('TestObject', (req) => {
+      expect(req.context.a).toEqual('a');
+    });
+    Parse.Cloud.afterDelete('TestObject', (req) => {
+      expect(req.context.a).toEqual('a');
+    });
+    // Save object
+    const obj = new TestObject();
+    await Parse.Object.saveAll([obj]);
+    await Parse.Object.destroyAll([obj], { context: { a: 'a' } });
+  });
+
+  it('should have access to context as destroy a object', async () => {
+    // Declare triggers
```

Comment:
Can you please remove these short comments, the code below is short and self-explanatory.

Reply:
done